pyanaconda/ui/gui/spokes/blivet_gui.py

```diff
@@ -21,10 +21,14 @@
 
 """Module with the BlivetGuiSpoke class."""
 from threading import Lock
-
-from blivetgui.osinstall import BlivetGUIAnaconda  # pylint: disable=import-error
-from blivetgui.communication.client import BlivetGUIClient  # pylint: disable=import-error
-from blivetgui.config import config  # pylint: disable=import-error
+from pyanaconda.errors import RemovedModuleError
+
+try:
+    from blivetgui.osinstall import BlivetGUIAnaconda  # pylint: disable=import-error
+    from blivetgui.communication.client import BlivetGUIClient  # pylint: disable=import-error
+    from blivetgui.config import config  # pylint: disable=import-error
```

Comment:
Pragmas are scoped, you should be able to put a single one on a line before the imports.